riva/proto/riva_nmt.proto

```diff
@@ -137,7 +137,7 @@ message TranslateTextRequest {
   string model = 2;
   string source_language = 3;
   string target_language = 4;
-
+  repeated string custom_dnt_phrases = 5;
```

Comment:
can we keep the field name as S2S/S2T cases for consistency? and also, let's please add a description of the field similar to below.

https://github.com/nvidia-riva/common/blob/main/riva/proto/riva_nmt.proto#L82

Reply:
Should I name the field as just "dnt_phrases"?

Reply:
yes we can name it as `dnt_phrases`